src/main/java/com/redhat/cloud/policies/engine/db/entities/AccountLatestUpdate.java

```diff
@@ -14,5 +14,8 @@ public class AccountLatestUpdate {
     @Column(name = "account_id")
     public String accountId;
 
+    @Column(name = "org_id")
+    public String orgId;
```

Comment:
This field is no longer needed here since tables are now separated. We need a new `OrgIdLatestUpdate` entity now.